src/web/app/pages-instructor/instructor-course-edit-page/instructor-edit-panel/instructor-edit-panel.component.html

```diff
@@ -72,14 +72,15 @@
         <input id="checkbox-display-instructor-{{ instructorIndex + 1 }}" checked=""
                type="checkbox" [ngModel]="instructor.isDisplayedToStudents"
                (ngModelChange)="triggerModelChange('isDisplayedToStudents', $event)" [disabled]="!instructor.isEditing">
+        &nbsp;
         <span ngbTooltip="If this is unselected, the instructor will be completely invisible to students. E.g. to give access to a colleague for ‘auditing’ your course">Display to students as:</span>
       </label>
       <div class="col-sm-9">
         <input id="displayed-name-instructor-{{ instructorIndex + 1 }}" class="form-control"
                type="text"
                (ngModelChange)="triggerModelChange('displayedToStudentsAs', $event)"
                [disabled]="!instructor.isDisplayedToStudents || !instructor.isEditing"
-               placeholder="E.g.Co-lecturer, Teaching Assistant"
+               placeholder="Instructor"
```

Comment:
It's not the placeholder; it's the default value that needs to be changed

Reply:
Isn't the default value the placeholder value? The corresponding issue states that it defaults to "Instructor" in the backend if left blank (which I assumed meant no input from user)

Reply:
`placeholder` is merely a displayed value in UI, not what is assigned in the data object.
Yes, doing this does resolve the issue in a way, but the input placeholder has another purpose which is to tell user what the field is about (when it is left empty).

Reply:
Yes, my understanding of #10444 was that we should display as the placeholder what the value would be if there was no input from the user, not to assign a default value since this is already performed in the backend. 

I agree that the placeholder should indicate what the field is about. Perhaps we could just extend the description to include "defaults to 'Instructor' if left blank". @damithc any issues with this alternative?

Reply:
> not to assign a default value since this is already performed in the backend

Back-end and front-end are separate entities. The things we do in the front-end are quality-of-life improvements, but the things we do in the back-end are sanity checks.

Reply:
We can either set the value to `Instructor` or add to the placeholder `(default: Instructor)`. I'm OK with either option.

Reply:
Okay I will do both update the placeholder and set a default value